src/LINEBot/Event/MessageEvent/MentioneeInfo.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+namespace LINE\LINEBot\Event\MessageEvent;
+
+class MentioneeInfo
```

Comment:
Could you change this name to `Mentionee`?
```suggestion
class Mentionee
```

Reply:
The emoji object is `EmojiInfo`, do you want to change it?

Reply:
If you change it, change here too.
`src/LINEBot/Event/MessageEvent/TextMessage.php` **line 59**

Reply:
Ah, it might be inconsistent...
Let's leave it then.